### PR TITLE
kpod attach

### PR DESCRIPTION
**Pull request by baude**
Attach to a running container

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: cmd/kpod/attach.go [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+)
+
+var (
+	attachFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "detach-keys",
+			Usage: "Override the key sequence for detaching a container. Format is a single character [a-Z] or ctrl-<value> where <value> is one of: a-z, @, ^, [, , or _.",
+		},
+		cli.BoolFlag{
+			Name:  "--no-stdin",
```
Comment (TomSweeneyRedHat) [inline cmd/kpod/attach.go]: I think you don't want the "--" at the start here.

File: cmd/kpod/attach.go [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+			Usage: "Do not attach STDIN. The default is false.",
+		},
+	}
+	attachDescription = "Attach to a running container"
```
Comment (rhatdan) [inline cmd/kpod/attach.go]: How about 
`The kpod attach command allows you to attach to a running container using the container's ID or name, either to view its ongoing output or to control it interactively. `

File: completions/bash/kpod [hunk 3]
```diff
@@ -653,6 +653,17 @@ __kpod_images() {
 	__kpod_q images $images_args | awk "$awk_script" | grep -v '<none>$'
 }
 
+_kpod_attach() {
```
Comment (rhatdan) [inline completions/bash/kpod]: You need to add attach below also in completions.

File: docs/kpod-attach.1.md [hunk 4]
```diff
@@ -1,31 +1,35 @@
...
 % Dan Walsh
-# kpod-attach "1" "September 2017" "kpod"
+# kpod-attach "1" "December 2017" "kpod"
 
 ## NAME
 kpod-attach - Attach to a running container
 
-## Description
+## SYNOPSIS
+**kpod detach [OPTIONS] CONTAINER**
```
Comment (rhatdan) [inline docs/kpod-attach.1.md]: s/detach/attach/g

File: docs/kpod-attach.1.md [hunk 5]
```diff
@@ -1,31 +1,35 @@
...
 
-1) Executing commands inside of the container
+**--no-stdin**
+Do not attach STDIN. The default is false.
 
-  We recommend that you use `kpod exec` to execute a command within a container
+## EXAMPLES ##
 
-  `kpod exec CONTAINERID /bin/sh`
+kpod attach foobar
```
Comment (rhatdan) [inline docs/kpod-attach.1.md]: Show a little output here to make it more real.

File: docs/kpod-attach.1.md [hunk 6]
```diff
@@ -1,31 +1,35 @@
...
-2) Viewing the output stream of the primary process in the container
+kpod attach 1234
 
-  We recommend that you use `kpod logs` to see the output from the container
-
-  `kpod logs CONTAINERID`
+kpod attach --no-stdin foobar
 
 ## SEE ALSO
 kpod(1), kpod-exec(1), kpod-logs(1)
```
Comment (rhatdan) [inline docs/kpod-attach.1.md]: Add kpod-run and remove kpod-logs.

File: cmd/kpod/attach.go (hunk 1 above)
Comment (baude) [reply cmd/kpod/attach.go]: fixed

File: cmd/kpod/attach.go (hunk 2 above)
Comment (baude) [reply cmd/kpod/attach.go]: fixed

File: completions/bash/kpod (hunk 3 above)
Comment (baude) [reply completions/bash/kpod]: fixed

File: docs/kpod-attach.1.md (hunk 4 above)
Comment (baude) [reply docs/kpod-attach.1.md]: fixed

File: docs/kpod-attach.1.md (hunk 6 above)
Comment (baude) [reply docs/kpod-attach.1.md]: fixed.

Comment (baude) [reply docs/kpod-attach.1.md]: done

File: docs/kpod-attach.1.md (hunk 5 above)
Comment (baude) [reply docs/kpod-attach.1.md]: fixed

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**